code/web/src/modules/user/api/state.test.js

```diff
@@ -0,0 +1,112 @@
+import Item from '../../subscription/Item';
+import userReducer from './state';
+
+describe('Reducer Testing', () => {
+  let initState;
```

Comment:
smart to set the initState outside the beforeEach hook so that it can be accessed later. Scope is still so hard for me to wrap my head around in JS

Reply:
It literally just clicked for me as we were working on this last night. @rwilliams659 came through with the `because scope` answer!